.github/actions/setup-job/action.yml

```diff
@@ -21,6 +21,8 @@ runs:
 
     - name: Install just
       uses: extractions/setup-just@aa5d15c144db4585980a44ebfdd2cf337c4f14cb #v1.4.0
+      env:
+        GITHUB_TOKEN: ${{ env.GITHUB_TOKEN }}
```

Comment:
Its still ab bit strange:

> this is because this workflow has to make requests to GitHub API in order to list available releases.

We explicitly define the exact version we want to use, it shouldn't make the requests. In the long run for improved supply chain security we should consider doing the same thing we do for cargo-sort, fetch the binary by known URL and known Hash.

Reply:
I gave a quick look, it does make the request and gets the download link from the response.